basiclibrary/src/main/java/basiclibrary/Lab3.java

```diff
@@ -0,0 +1,59 @@
+package basiclibrary;
+
+import java.util.*;
+
+public class Lab3 {
+
+    public String tempFinder(int[][] inputArr) {
+        int min = 100;
```

Comment:
It's standard in Java to use all caps when storing a hard coded value:
MIN = 100;
MAX = 0;

If the variable name has more than one word, separate each word with a _:
MIN_VALUE = 100;

Reply:
Got it. I'll try to remember this going forward.